dev-packages/localization-manager/src/deepl-api.ts

```diff
@@ -40,9 +41,38 @@ export async function deepl(
     for (const response of responses) {
         mergedResponse.translations.push(...response.translations);
     }
+    for (const translation of mergedResponse.translations) {
+        translation.text = coerceTranslation(translation.text);
+    }
     return mergedResponse;
 }
 
+/**
+ * Coerces the target language into a form expected by Deepl.
+ *
+ * Currently only replaces `ZH-CN` with `ZH`
+ */
+function coerceLanguage(parameters: DeeplParameters): void {
+    if (parameters.target_lang === 'ZH-CN') {
+        parameters.target_lang = 'ZH';
+    }
```

Comment:
Does DeepL allow other `ZH` variants (`ZH-TW`, e.g.), or does it expect all `ZH` to be only `ZH`?